app/services/melanges/update_melange.rb

```diff
@@ -0,0 +1,26 @@
+module Melanges
+  class UpdateMelange < ApplicationService
+    def call
+      update_melange
+    end
+
+    private
+
+    def initialize(melange, melange_params)
+      @melange = melange
+      @melange_params = melange_params
+    end
+
+    def update_melange
+      Melange.transaction do
+        @melange.assign_attributes(@melange_params)
+        if @melange.valid?
+          @melange.save!
```

Comment:
save! i tak zwróci true, więc zwracanie tego niżej jest zbędne, no a jeśli coś się nie powiedzie, to i tak poleci 500 więc niżej true nie zostanie wykonane